src/python/foglamp/device/http_south_device.py

```diff
@@ -0,0 +1,188 @@
+# -*- coding: utf-8 -*-
+
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+"""HTTP Listener handler for sensor readings"""
+import sys
+from aiohttp import web
+import asyncio
+from foglamp import logger
+from foglamp.web import middleware
+from foglamp.device.ingest import Ingest
+
+__author__ = "Amarendra K Sinha"
+__copyright__ = "Copyright (c) 2017 OSIsoft, LLC"
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+_LOGGER = logger.setup(__name__)
+
+_CONFIG_CATEGORY_NAME = 'http_south'
```

Comment:
Note that this should be consistent with the original category name HTTP_SOUTH. However, currently server.py currently does not utilize this part of plugin_info (which is an existing issue). COAP suffers the same.